public/sass/base/font-awesome/_variables.scss

```diff
@@ -8,7 +8,7 @@ $fa-line-height-base: 1 !default;
 $fa-css-prefix: fa !default;
 $fa-version: '4.7.0' !default;
 $fa-border-color: #eee !default;
-$fa-inverse: #fff !default;
+$fa-inverse: #ffffff !default;
```

Comment:
Why do we need to make this change?  It seems like `#fff` (and `#eee` above it) was previously legal.  The error obviously indicates one of them is incorrect, but changing all of them will possibly break other things.

Reply:
#fff is legal but its not legal to the react-monaco-editor apparently since grafana removed tslib from the toolkit from [Grafana v8.3.4](https://github.com/grafana/grafana/releases/tag/v8.3.4). [PR #43556](https://github.com/grafana/grafana/pull/43556). $fa-inverse specifically wasn't causing an issue so i reverted it, but the palette white has to be #FFFFFF for it to not show errors